src/Microsoft.AspNetCore.Server.IISIntegration.Tasks/WebConfigTransform.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration.Tasks
+{
+    internal static class WebConfigTransform
```

Comment:
To reviewer: this is an exact copy of already-reviewed code that was in dotnet-publish-iis